README.md

```diff
@@ -47,6 +48,17 @@
 
 - **[add score]**
 - **[refresh score]**
+- **[unique design]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://memicelis.github.io/Leaderboard/dist)
```

Comment:
Great job on designing this page, however your live link styling is not updating, kindly fix that or remove live demo link.